Lib/test/test_asyncio/test_server.py

```diff
@@ -147,6 +149,74 @@ async def serve(*args):
         await task2
 
 
+class UnixServerCleanupTests(unittest.IsolatedAsyncioTestCase):
+    @socket_helper.skip_unless_bind_unix_socket
+    async def test_unix_server_addr_cleanup(self):
```

Comment:
Would you mind adding one-line comments to the new tests explaining what each test checks? They look so similar I first thought you had accidentally copied the test a few times. :-)

Reply:
Sure. A commit with comments have been pushed.